public/static/locales/isl/l2.json

```diff
@@ -1,8 +1,8 @@
 {
     "sign-in-hello": "Þú getur fylgst með framvindu þinni með því að búa til aðgang.",
     "sign-in-hello-link": "Smelltu hér til að skrá þig inn eða búa til aðgang.",
-    "l2-explanation-long": "Þessi söfnun hefur sem markmið að auka fjölbreytni í íslenskum máltæknigögnum. Máltækni hefur náð langt á undanförnum árum en mörg talgreiningarkerfi eiga enn erfitt með að skilja íslensku innflytjenda. Við hvetjum alla sem eiga íslensku sem annað tungumál að taka þátt, jafnt byrjendum sem þeim sem tala reiprennandi íslensku. Þetta safn ætti að ná yfir alls kyns íslensku.",
+    "l2-explanation-long": "Þessi söfnun hefur sem markmið að auka fjölbreytni í íslenskum máltæknigögnum. Máltækni hefur náð langt á undanförnum árum en mörg talgreiningarkerfi eiga enn erfitt með að skilja íslensku innflytjenda. Við hvetjum alla sem hafa íslensku sem annað tungumál að taka þátt, jafnt byrjendum sem og þeir sem tala reiprennandi íslensku. Þetta safn ætti að ná yfir alls kyns íslensku.",
```

Comment:
byrjendum -> byrjendu**r**

Reply:
Æi yep good catch! Fixed in [9b6d763](https://github.com/cadia-lvl/samromur/pull/153/commits/9b6d763dcc19d3e353a23861ce97322aba2cc9aa)